app/javascript/react/components/ParkIndexTile.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const ParkIndexTile = props => {
```

Comment:
I think these names could be better. Using `Index` and `Show` in the component name automatically "couples" the component to the relevant view. Could you use a `ParkTile` and a `DetailedParkTile`?